### ISSUE TITLE
Separate field for cookie description

### ISSUE THREAD

**Issue by WHITE-developer** [nilsenpaul/craft-complete-cookie-consent#26]
Would be nice to have a separate field for the cookie description so you can do some additional styling on it like a smaller font.

**Comment by nilsenpaul** [comment]
This has been fixed in v1.0.6.

**Comment by WHITE-developer** [comment]
Let me rephrase it. What I meant was to split `Name` into `Name` and `Description`, so you can add some more info what the cookie group is about.
We are now doing some additional js to accomplish it, but would be great to have it in separate fields.
![Screenshot from 2021-09-15 11-12-18](https://user-images.githubusercontent.com/22771556/133406110-71403856-3aa0-4bde-8bb2-913c9ab160b1.png)

**Comment by nilsenpaul** [comment]
Ah, so THAT's what you meant ;)

We'd have to add a seperate description column for that, shouldn't be a problem.

**Comment by nilsenpaul** [comment]
We've just tagged v1.0.9.3, giving you the option to add a description to cookie types (and having those displayed in the cookie banner).

**Comment by WHITE-developer** [comment]
Works perfect in the banner. But in twig I get `Key "description" for array with keys "name, defaultOn, required" does not exist.`, can you add it to there too? I think in models/CookieSettings.php line 46

**Comment by nilsenpaul** [comment]
Consider it fixed.
